src/models/posts.models.js

```diff
@@ -0,0 +1,33 @@
+import { v4 as uuidv4 } from 'uuid';
+
+let posts = [
+	{
+		author: 'Jules NTARE',
+		title: 'JS new features',
+		desc: 'lorem ipsum',
+		'cover-imgUrl':
+			'https://firebasestorage.googleapis.com/v0/b/portfolio-db-d0d5c.appspot.com/o/images%2Ftimerate.PNG?alt=media&token=2fda0eb1-7f43-4e87-b2b5-1396c3d40b5e',
```

Comment:
It is not safe to expose your token in your codebase. You should use [DotEnv npm package](https://www.npmjs.com/package/dotenv)  to add the token via environment variable